source/stylesheets/screen_assessment.css.scss

```diff
@@ -2,6 +2,6 @@
 
 // Here we inject page-specific variable values:
 $nav-subitem-bg: #4AAAD3;
-$active-header-bg: #3086AA;
+$nav-active-parent-bg: #3086AA;
```

Comment:
Thank you for going beyond the scope of the card to fix this color issue! Across all the different API docs, too!